src/main/resources/interface-specification.yml

```
@@ -1496,6 +1496,48 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  '/keys/{kId}':
+    get:
+      tags:
+        - gateway
+      summary: Retrieve key's JWK
+      description: Retrieve the JWK of the key with Id equal to kId
+      operationId: getKeyJWKfromKId
```

`JWK` is one of the formats a key could be represented with, not a property.
A more suitable description could be
```suggestion
      summary: Retrieve key in JWK format
      description: Retrieve the JWK by kid
      operationId: getJWKByKid
```

Note: despite meaning "key Id", the standard naming convention by RFC is `kid` instead of `kId`